src/scripts/create_Drop-seq_reference_metadata.sh

```diff
@@ -60,21 +60,21 @@ set -e
 
 
 
-while getopts ":n:r:s:g:f:o:a:i:b:evh" options; do
+while getopts ':n:r:s:g:f:o:a:b:i:veh' options; do
   case $options in
     n ) reference_name=$OPTARG;;
     r ) reference_fasta=$OPTARG;;
     s ) species=$OPTARG;;
     g ) gtf=$OPTARG;;
-    f ) filtered_gene_biotypes="$filtered_gene_biotypes G=$OPTARG";;
+    f ) filtered_gene_biotypes="G=$OPTARG";;
```

Comment:
Good catch! I'll fix that tomorrow. Awaiting your response to the other point to hopefully fix everything right away.

I'll convert this PR to a draft until the to prevent accidentally merging it before I fix this.